From 0eff41601723e51d975af4e4663dc788bb067f0a Mon Sep 17 00:00:00 2001
From: Jerry Sun <jerry.sun@windriver.com>
Date: Mon, 16 Jan 2023 11:35:58 -0500
Subject: [PATCH 1/1] Add permissive cluster policy by default

The default permissive policy is only added if the policy set is
set to IKS. In the non IKS case, no default cluster image policy
is created. This means all pod deployment is denied in all
namespaces that don't have any imagepolicies. This patch applies
the default permissive cluster image policy to not interfere with
platform functionality when Portieris is applied.

Signed-off-by: Jerry Sun <jerry.sun@windriver.com>
---
 helm/portieris/templates/policies.yaml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/helm/portieris/templates/policies.yaml b/helm/portieris/templates/policies.yaml
index 6e67ffc..ed8c84f 100644
--- a/helm/portieris/templates/policies.yaml
+++ b/helm/portieris/templates/policies.yaml
@@ -65,7 +65,6 @@ spec:
 ---
 {{ end }}
 
-{{- if (eq .Values.PolicySet "IKS") }}
 apiVersion: portieris.cloud.ibm.com/v1
 kind: ClusterImagePolicy
 metadata:
@@ -77,4 +76,3 @@ spec:
    repositories:
    {{- .Values.clusterPolicy.allowedRepositories | toYaml | nindent 4 }}
 ---
-{{ end }}
-- 
2.34.1

